pkgs/top-level/all-packages.nix

```diff
@@ -34698,6 +34698,18 @@ with pkgs;
 
   super-slicer-latest = super-slicer.latest;
 
+  bambu-studio = callPackage ../applications/misc/bambu-studio {
+    inherit (gst_all_1) gstreamer gst-plugins-base gst-plugins-bad;
+
+    glew = glew-egl;
+
+    wxGTK31 = wxGTK31.override {
+      withCurl = true;
+      withPrivateFonts = true;
+      withWebKit = true;
```

Comment:
With `wxGTK32` `withWebKit=true;` will be default from now on

https://github.com/NixOS/nixpkgs/pull/268348/files